core/templates/css/oppia.css

```diff
@@ -3915,11 +3915,11 @@ div.oppia-issues-learner-action {
   are used for correctly stacking AngularJS modals on top of Angular modals.
 */
 .forced-modal-stack-over {
-  z-index: 1060 !important;
+  z-index: 1080 !important;
```

Comment:
@srijanreddy98 I went ahead with this (non-ideal) solution for now because the fix is kind of urgent for users. As discussed, the long term fix for it would be to migrate the rte-helper service.